packages/sourcecred/src/core/ledger/ledger.js

```diff
@@ -518,7 +531,13 @@ export class Ledger {
     if (!this._accounts.has(id)) {
       throw new Error(`identity ${id} not found`);
     }
-    const {active} = this.account(id);
+    const {active, payoutAddresses} = this.account(id);
+    if (this._externalCurrency) {
+      const currencyKey = getCurrencyKey(this._externalCurrency);
+      if (!payoutAddresses.get(currencyKey)) {
+        throw new Error("Cannot activate user: No payout address set");
+      }
+    }
```

Comment:
Would you put this in _activate?

Reply:
@blueridger I'm not sure I understand why. We only need to check this during the initial activation attempt. Thereafter, can't we rest assured that the prior state leading up to the activation will always be the same? Isn't that kinda the point of this data structure?